tekton/cronjobs/README.md

```diff
@@ -213,11 +213,13 @@ spec:
               value: tekton/images/myimage
 ```
 
-4. Add PLATFORMS to `cronjob.yaml` if you want to build multi-arch image. List target architectures for the image.
+4. Add PLATFORMS and BUILD_TYPE to `cronjob.yaml` if you want to build multi-arch image. List target architectures for the image. For BUILD_TYPE use `docker` or `ko` value to choose the image build tool
```

Comment:
NIT: missing dot at the end. Perhaps you could add a link to the template used for each of `ko` and `docker`?

Reply:
fixed and added. Thank you